docs/HowToUseSmartthingsSDK.md

```diff
@@ -0,0 +1,23 @@
+How to use SmartThings SDK
+===================
+
+Lightweight OCF development framework based Tizen RT.
+
+----------
+
+
+Introduction
+-------
+SmartThings SDK supports Samsung Connect Services - provided by Samsung mobile devices. Through the SmartThings Console, you can easily connect your device to the cloud. It also provides an environment to test the developed automation app.
+
+How to use
+------
+####1. Using Sample App
```

Comment:
1. Let's remove "####".
2. Add two spaces at end of line.

Reply:
Ok